src/property.ts

```diff
@@ -1080,6 +1081,7 @@ function Property$setValue(property: Property, obj: Entity, currentValue: any, n
 			var eventArgs: PropertyChangeEventArgs = { entity: obj, property, newValue, oldValue };
 			(property.changed as EventPublisher<Entity, PropertyChangeEventArgs>).publish(obj, additionalArgs ? merge(eventArgs, additionalArgs) : eventArgs);
 			(obj.changed as Event<Entity, EntityChangeEventArgs>).publish(obj, { entity: obj, property, oldValue, newValue });
+			property.containingType.model.afterPropertySet.publish(obj, { entity: obj, property, newValue, oldValue });
```

Comment:
It may not really matter, but it makes more sense to me for the order of events to be the same in all cases. Here, the global event is published _after_ the object event, but for lists, the global event is published _before._

Reply:
Moved to the top for both cases.